first version of themed views completed

cyco77 opened the pull request:
- Connection Details
- Entity Details
- Entity List
- Smart Match untouched, because complex situation around list control

Power-Maverick commented:
Looks like the vscode webview ui package you have referenced got deprecated. [link](https://github.com/microsoft/vscode-webview-ui-toolkit/issues/561#issuecomment-2573702014)

cyco77 commented:
will check it

cyco77 commented:
I removed the deprecated toolkit and replaced it with pure css... not as fancy as the toolkit, but I think it looks nice... In the next couple of days I play with react in vscode extensions... maybe this could make it much easier to maintain and extend.

Power-Maverick commented:
I'll test it out in next few days. If everything is working fine I'll release a new version. Thanks for the PR and improving the tool.
